src/common/config.cc

```diff
@@ -641,9 +641,6 @@ int md_config_t::parse_argv(ConfigValues& values,
     else if (ceph_argparse_witharg(args, i, &val, "--show_config_value", (char*)NULL)) {
       do_show_config_value = val;
     }
-    else if (ceph_argparse_flag(args, i, "--no-mon-config", (char*)NULL)) {
-      values.no_mon_config = true;
-    }
```

Comment:
this probably shouldn't get removed?

Reply:
yeah, not sure how that happened. I may have misread that as the `no-config-file` and dropped it.